src/main/java/com/brvsk/ZenithActive/course/CourseServiceImpl.java

```diff
@@ -66,6 +74,32 @@ public List<CourseResponse> getCoursesForInstructor(UUID instructorId) {
                 .collect(Collectors.toList());
     }
 
```

Comment:
i know its Set but its good to check if member is not enroled and thorow en error if it is enrolled to course